content/docs/operating/security.md

```diff
@@ -161,3 +161,9 @@ members of the Prometheus development team and the staff of those providers
 have access. If you are concerned about the exact provenance of your binaries,
 it is recommended to build them yourself rather than relying on the
 pre-built binaries provided by the project.
+
+## External audits
+
+CNCF sponsored an external security audit by cure53 which ran from April 2018
```

Comment:
I would link both the CNCF and cure53.

Reply:
I thought they would be obvious once you are this deep in our pages, but will do.